js/src/configuration-wizard.js

```diff
@@ -31,8 +31,9 @@ class App extends React.Component {
 
 		this.state = {
 			isLoading: true,
+			// Use the hash from the url without the hashtag.
+			currentStep: window.location.hash.substring( 1 ),
```

Comment:
Is there a specific reason to have this (`currentStep`) in the state of this component? It seems that it is only needed for a one time situation.